core/src/de/golfgl/gdxgamesvcs/IGameServiceListener.java

```diff
@@ -27,7 +31,19 @@
      * @param msg further information, may be null
      * @param t   Throwable causing the problem, may be null
      */
-    public void gsShowErrorToUser(GsErrorType et, String msg, Throwable t);
+    public void gsShowErrorToUser(GsResultCode et, String msg, Throwable t);
 
-    public enum GsErrorType {errorLoginFailed, errorUnknown, errorServiceUnreachable, errorLogoutFailed}
```

Comment:
Rename GsErrorType to GsResultCode. Makes it usable in success cases also. Relevant to for example on session change events.

Reply:
What different success types can we have?

Reply:
Instead of defining only some generic error types for example `errorLoginFailed `or `errorLogoutFailed` which we use to inform the client when an error occurs, we could define some other status codes as well like `signedIn`, `signedOut`, `connectionPaused`, `connectionResumed `etc and rename the enum type to `GsResultCode`. We could then use it with `OnSessionAcrive` and `OnSessionInactive` methods to inform the client the reason why the session state changed.

Does this idea sound reasonable or is this completely useless thing to consider. What do you think?

Reply:
Not useless, but at the state GPGS was there was no reliable information what exactly happened, might have changed now.